### DISCUSSION TITLE
Signalr 5.x -- implement NTLM using Java client

### DISCUSSION THREAD

**Discussion by zulander1** [dotnet/aspnetcore#28599] [General]
Is there a way to implement NTML authentication using username password for Signalr core 5.x

**Comment by BrennanConroy** [comment]
You can modify the `OkHttpClient.Builder` when setting up the SignalR connection using `setHttpClientBuilderCallback(builder -> {});`

A quick search showed an issue on OkHttp with people sharing code to get NTLM auth working https://github.com/square/okhttp/issues/206
